src/Microsoft.AspNet.Hosting/WebHostBuilderExtensions.cs

```diff
@@ -10,102 +10,102 @@
 
 namespace Microsoft.AspNet.Hosting
 {
-    public static class WebApplicationBuilderExtensions
+    public static class WebHostBuilderExtensions
```

Comment:
I think @Tratcher wanted WebHostServiceRunnerExtensions, because of potential conflicts?